LiskHQ/lisk-mobile
# Use Node.js version from .nvmrc file on Jenkins - Closes #255

# What was the bug or feature?
Whatever version of Node.js was on the Jenkins build node was being used uncontionally.

### How did I fix it?
Added a `.nvmrc` file that gets used is the `Jenkinsfile` to decide what Node.js version to use.

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
Run build on Jenkins, verify that it passes and that the expected Node.js version is being used.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
